CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ Changes in this release:
 - Define explicit path in protocol methods
 - Added support for schema management for multiple slices in the same database (#1207)
 - Marked pypi package as typed
+- Added support for /etc/inmanta.d style configuration files (#183)
+
+DEPRECATIONS:
+* The files /etc/inmanta/agent.cfg and /etc/inmanta/server.cfg are not used anymore. More information about the available 
+configuration files can be found in the documentation pages under `Administrator Documentation -> Configuration files`.  
```

Comment:
The rpm should be able to correctly move the existing config file, so that the update doesn't break existing installs